app/javascript/mastodon/features/ui/components/media_modal.js

```diff
@@ -211,7 +211,6 @@ class MediaModal extends ImmutablePureComponent {
             style={swipeableViewsStyle}
             containerStyle={containerStyle}
             onChangeIndex={this.handleSwipe}
-            onSwitching={this.handleSwitching}
```

Comment:
Unrelated, but ok.

Reply:
Yes, turns out this was never defined/used.